doc.go

```diff
@@ -58,6 +58,7 @@ Application Options:
       --proxy=             Connect via SOCKS5 proxy (eg. 127.0.0.1:9050)
       --proxyuser=         Username for proxy server
       --proxypass=         Password for proxy server
+      --proxyrandomize     Randomize user credentials for each proxy connection. This enables Tor stream isolation
```

Comment:
This doesn't match how it's actually shown.  Please use `./btcd -h` and copy it from there so this matches.